### PR TITLE
Schema validation

### PR DESCRIPTION
**Pull request by Fatmasiam**
#74

### CODE REVIEW

File: src/helpers/validation-shemas.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  lastname: joi.string().alphanum().min(3).max(30)
+    .required(),
+  mobile_number: joi.string().required(),
+  email: joi.string().email({
+    minDomainAtoms: 2,
+  }).required(),
+  specalization_id: joi.number().integer().required(),
+  freelancer_url: joi.string().regex(/http(s)?:\/\/([\w]+\.)?[A-z0-9_-]+\.com\/[A-z0-9_-]+/).required(),
+  photo_url: joi.string().regex(/^https?:\/\/([\w]||[-_.])+\.[a-z]{2,4}(\/||[\w-_])*\.(png|jpe?g||gif)$/).required(),
+  password: joi.string().required(),
```
Comment (tomduggan85) [inline src/helpers/validation-shemas.js]: @Fatmasiam Maybe `password` could have a min() also

Comment (Fatmasiam) [reply src/helpers/validation-shemas.js]: and regex also,  I will make it 
thanx

Comment (tomduggan85) [reply src/helpers/validation-shemas.js]: @Fatmasiam Cool!

**Review by ahmedalmi95 — APPROVED** [review]

File: src/helpers/validation-shemas.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const joi = require('joi');
+
+exports.signUpSchema = joi.object().keys({
+  firstname: joi.string().alphanum().min(3).max(30)
+    .required(),
+  lastname: joi.string().alphanum().min(3).max(30)
```
Comment (Angham116) [inline src/helpers/validation-shemas.js]: ```firstname``` and ```lastname``` should be a string with alpha only .remove ```alphanum()``` from here.
and use regex with chars [a-z and A-Z]

**Review by Angham116 — APPROVED** [review]

File: src/helpers/validation-shemas.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const joi = require('joi');
+
+exports.signUpSchema = joi.object().keys({
+  firstname: joi.string().regex(/^[a-zA-Z]+$/).min(3).max(30)
+    .required(),
+  lastname: joi.string().regex(/^[a-zA-Z]+$/).min(3).max(30)
+    .required(),
+  mobile_number: joi.string().required(),
```
Comment (kannanhassouna) [inline src/helpers/validation-shemas.js]: mobile_number should have at least 10 characters.
#21

**Comment by kannanhassouna** [comment]
@Fatmasiam 
do you think the name of the branch is correct ?
and the file name incorrect, that should be ``` validation-schemes ```

**Review by kannanhassouna — APPROVED** [review]

**Pull request merged**